tests/cache/cdn2.thecatapi.com.images-cjd.jpg.php

```diff
@@ -0,0 +1,97 @@
+<?php
+declare(strict_types = 1);
+
+return [
+    'headers' => [
+        'date' => [
+            'Fri, 11 Dec 2020 13:05:24 GMT'
+        ],
+        'content-type' => [
+            'image/jpeg'
+        ],
+        'content-length' => [
+            '115080'
+        ],
+        'last-modified' => [
+            'Thu, 27 Sep 2018 01:08:20 GMT'
+        ],
+        'etag' => [
+            '"cf16a912a37899c395d13e4365cddc66"'
+        ],
+        'x-cache' => [
+            'Miss from cloudfront'
+        ],
+        'via' => [
+            '1.1 efe54e8b68e074d39b2ecd249f85100b.cloudfront.net (CloudFront)'
+        ],
+        'x-amz-cf-pop' => [
+            'HIO50-C1'
+        ],
+        'x-amz-cf-id' => [
+            'CqL0begLlLoNbhHZ6fz-yYoddZh28AwHJqOWMjmxCvv7CH5Xhpb9RA=='
+        ],
+        'cache-control' => [
+            'max-age=14400'
+        ],
+        'cf-cache-status' => [
+            'HIT'
+        ],
+        'age' => [
+            '418'
+        ],
+        'accept-ranges' => [
+            'bytes'
+        ],
+        'cf-request-id' => [
+            '06f3810d3c0000416edc1e1000000001'
+        ],
+        'expect-ct' => [
+            'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"'
+        ],
+        'report-to' => [
+            '{"endpoints":[{"url":"https:\\/\\/a.nel.cloudflare.com\\/report?s=8g481Ul3yJLvZOkfPwoQOhiWAkriLSSrLQyONKLJqyYlmSj3uZSap8Xb4Xsqeqmztf%2B5P5jP%2BBMUq80in1XMOmB0kbaCy4G0mfND9kA6ttyJWs0%3D"}],"group":"cf-nel","max_age":604800}'
+        ],
+        'nel' => [
+            '{"report_to":"cf-nel","max_age":604800}'
+        ],
+        'vary' => [
+            'Accept-Encoding'
+        ],
+        'server' => [
+            'cloudflare'
+        ],
+        'cf-ray' => [
+            '5fff6ac1f832416e-HAM'
+        ],
+        'Content-Location' => [
+            'https://cdn2.thecatapi.com/images/cjd.jpg'
+        ],
+        'X-Request-Time' => [
+            '0.048 ms'
+        ]
+    ],
+    'statusCode' => 200,
+    'reasonPhrase' => 'OK',
+    'body' => '����' . "\0" . 'JFIF' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '' . "\0" . '��' . "\0" . 'C' . "\0" . '	
```

Comment:
In theory, the response shouldn't contain any data in the body, if it's binary.

Reply:
Wasn't sure about this either, but it's not the whole body and just the first bytes. Maybe the first data that is loaded before the progress function aborts loading further data.